example/panel/modules/data_load_async_module.dart

```diff
@@ -42,8 +44,10 @@ class DataLoadAsyncModule extends Module {
   @protected
   Future<Null> onLoad() {
     // trigger non-blocking async load of data
+    _events.didLoadData.first
+        .then((_) => specifyStartupTiming(StartupTimingSpecifier.firstUseful));
     _actions.loadData();
-    return new Future.value();
+    return null;
```

Comment:
This could introduce an issue if a consumer treats the returned `Future` directly.

```dart
// This will throw.
_module.onLoad().then((_) => ...);
```

Reply:
I was just toying around with the examples and forgot to put it back.